website/client/components/userMenu/inbox.vue

```diff
@@ -353,6 +353,9 @@ export default {
     },
     toggleOpt () {
       this.$store.dispatch('user:togglePrivateMessagesOpt');
+    },
+	async sendMessageShortcut () {
```

Comment:
Weird indentation here.

Reply:
this + this function isn't really necessary as it just calls another one, better use `@keyup.ctrl.enter='sendPrivateMessage()'`